Help center article for privacy settings (#38)
famedly/helpcenter: docs/help-center/using-famedly/login.mdx

---
sidebar_label: Login
title: Login
hide_title: true
displayed_sidebar: helpcenterSidebar
sidebar_position: 1
---

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';
import useBaseUrl from '@docusaurus/useBaseUrl';

<div class="hero hero--primary">
  <div class="container">
    <h2 class="hero__title">Login</h2>
    <p class="hero__subtitle">How to log in to Famedly and what settings can be made on login</p>
  </div>
</div>

## Log in

<Tabs>
  <TabItem value="desktop" label="Desktop" default>
  <ol>
    <li>Open Famedly web app.</li>
    <li>Enter your matrix servers URL in the input field <b>Login URL</b>.</li>
    <li>Click on <b>Next</b>.</li>
    <li>Enter your username in the input field <b>Username</b> and your password in the input field <b>Password</b>.</li>
    <li>Click on <b>Login</b>.</li>
  </ol>
  </TabItem>
  <TabItem value="mobile" label="Mobile">
  <ol>
    <li>Open Famedly app.</li>

>     <li>Enter your matrix servers URL in the input field <b>Login URL</b>.</li>

Reviewer: ```suggestion
    <li>Enter your login URL into the corresponding input field.</li>
```
